packages/subscriber/src/components/add-form/style.scss

```diff
@@ -200,6 +200,7 @@
 			color: var(--color-text);
 			text-decoration: underline;
 			height: unset;
+			font-size: 0.875rem;
```

Comment:
Why this value, and not a variable from the shared typography package?

https://github.com/Automattic/wp-calypso/blob/e7e2d47f075b46e1ddb875f5b4cd0576b8e1f800/packages/typography/styles/variables.scss#L46

The same in higher up.

If you just remove these lines, does it actually get defaulted to 14px?

Reply:
Thanks, changed it to use the variable in both places👌 

I tried removing the font-size and the line-height, but link becomes 13px and text 16px